### PR TITLE
MGDCTRS-1793 : Introducing Dataplane Operator dashboard on Stage env

### PR DESCRIPTION
**Pull request by rpancham**
Introducing Dataplane Operator dashboard on Stage env

## Checklist (Definition of Done)

- [ ] All acceptance criteria specified in JIRA have been completed
- [ ] Unit and integration tests added that prove the fix is effective or the feature works (tested against emulated and non-emulated OCM environment)
- [ ] Verified independently by reviewer
- [ ] Required Standard Operating Procedure (SOP) is added.

## Prometheus

- [ ] Changes were tested against dev environment

## Grafana

- [ ] You have imported the changes into a dev or staging environment

### CODE REVIEW

File: resources/prometheus/downstream/grafana-dashboard-dataplane-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,1441 @@
+---
+annotations:
```
Comment (vaibhavjainwiz) [inline resources/prometheus/downstream/grafana-dashboard-dataplane-operator.yaml]: Add configmap content. Check How other dashboard file are written

**Review by vaibhavjainwiz — CHANGES_REQUESTED** [review]

File: resources/prometheus/downstream/grafana-dashboard-dataplane-operator.yaml (hunk 1 above)
Comment (rpancham) [reply resources/prometheus/downstream/grafana-dashboard-dataplane-operator.yaml]: @vaibhavjainwiz  Added the configmap content

**Comment by Srihari1192** [comment]
@rpancham Please find the observation
When hovering the particular chart, dotted cross lines is displaying for all the graph instead of the particular graph
Screenshot from 2023-04-17 13-26-18

**Comment by rpancham** [comment]
> @rpancham Please find the observation When hovering the particular chart, dotted cross lines is displaying for all the graph instead of the particular graph Screenshot from 2023-04-17 13-26-18

Working on this @Srihari1192

**Comment by rpancham** [comment]
> @rpancham Please find the observation When hovering the particular chart, dotted cross lines is displaying for all the graph instead of the particular graph Screenshot from 2023-04-17 13-26-18

@Srihari1192 I guess the issue is not from our end it might be configuration issue but i guess it should not be the blocker for merge

**Comment by rpancham** [comment]
Following up this in the new PR with new branch

**Pull request closed without merging**